scripts/ai/jobs/CpJobParameters.lua

```diff
@@ -178,6 +178,17 @@ function CpFieldWorkJobParameters:lessThanThreeMultiTools()
     return self:getMultiTools() < 4
 end
 
+function CpFieldWorkJobParameters:onLaneOffsetChanged(setting)
+    local vehicle = self.job:getVehicle()
+    if vehicle then
+        local course = vehicle:getFieldWorkCourse()
+        if course and course:getMultiTools() > 1 then
+            vehicle:getFieldWorkCourse():setPosition(vehicle:getCpLaneOffsetSetting():getValue())

```

Comment:
Why go to the vehicle:getCpLaneOffsetSetting(), laneOffset is a member of this class.